src/main/java/com/google/sps/servlets/utils/CommentAnalysis.java

```diff
@@ -53,12 +53,26 @@ public CommentAnalysis(LanguageServiceClient languageService) {
    * @return a Statistics object that contains required values to display
    */
   public Statistics computeOverallStats(CommentThreadListResponse youtubeResponse) {
-    List<Double> scoreValues =
+    List<Sentiment> documentList =
         youtubeResponse.getItems().stream()
             .map(UserComment::new)
-            .map(this::calcualateSentiAnalysisScore)
+            .map(
+                comment ->
+                    languageService
+                        .analyzeSentiment(
+                            Document.newBuilder()
+                                .setContent(comment.getCommentMsg())
+                                .setType(Document.Type.PLAIN_TEXT)
+                                .build())
```

Comment:
Nit: should these be indented 4 spaces each?

Reply:
no idea why this auto checkstyle makes it like this hhh